src/workers/event.rs

```diff
@@ -114,15 +125,42 @@ where
         {
             trace!(actor = ?current, "Found existing actor data");
             // Merge in current counts
-            actor_data.count = current.count;
+            actor_data.instances = current.instances;
+        }
+        // Update actor count in the host
+        if let Some(mut host) = self.store.get::<Host>(lattice_id, &actor.host_id).await? {
+            trace!(host = ?host, "Found existing host data");
+            match host.actors.get(&actor.public_key) {
```

Comment:
Nit: Consider using the [entry api](https://doc.rust-lang.org/1.68.2/std/collections/hash_map/struct.HashMap.html#method.entry) here instead

Reply:
FINE. I have to clone but I'll do it